feature/client/src/commonMain/kotlin/com/mifos/feature/client/clientIdentitiesList/ClientIdentitiesListScreen.kt

```diff
@@ -128,11 +124,12 @@ internal fun ClientIdentitiesListScreen(
                                     id = if (item.id != null) item.id.toString() else emptyMessage,
                                     key = item.documentKey ?: emptyMessage,
                                     status = if (item.status != null) {
-                                        if (item.status!!.lowercase().endsWith("active")) Status.Active
-                                        if (item.status!!.lowercase()
-                                                .endsWith("inactive")
+                                        if (item.status!!.lowercase().endsWith("inactive")) {
```

Comment:
cashing the status should be moved to utils function, please move it as util function.